blog/zh/blog/2023/10/09/wecity-uses-apisix.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "荷兰智慧城市革命：WeCity 与 APISIX 的合作之旅"
+authors:
+  - name: Arjen Hof
+    title: Author
+    url:
+    image_url:
+  - name: Tim van Densen
+    title: Author
+    url:
+    image_url:
+  - name: Jing Yan
+    title: Technical Writer
+    url:
+    image_url: https://avatars.githubusercontent.com/u/114121331?v=4
```

Comment:
For `image_url`, if you do `https://avatars.githubusercontent.com/u/114121331?v=4`, it won't change if you update your github profile pic. On the other hand, if you do `https://github.com/JYan00.png`, it automatically updates when you change your profile pic,

```suggestion
  - name: Jing Yan
    title: Technical Writer
    url: https://github.com/JYan00
    image_url: https://github.com/JYan00.png
```

Reply:
> For `image_url`, if you do `https://avatars.githubusercontent.com/u/114121331?v=4`, it won't change if you update your github profile pic. On the other hand, if you do `https://github.com/JYan00.png`, it automatically updates when you change your profile pic,

Thanks for reminding!